rmgpy/cantherm/statmech.py

```diff
@@ -993,3 +1048,59 @@ def projectRotors(conformer, F, rotors, linear, TS):
         
     return numpy.sqrt(eig[-Nvib:]) / (2 * math.pi * constants.c * 100)
 
+def assign_frequency_scale_factor(model_chemistry):
+    """
+    Assign the frequency scaling factor according to the model chemistry.
+    Sources: http://cccbdb.nist.gov/vibscalejust.asp, J. Chem. Phys. 1999, 110, 2822–2827
```

Comment:
One thing to think about going forward is which values we want to assign as default. For example, [this website](https://comp.chem.umn.edu/freqscale/index.html) seems to be more actively maintained, but it's unclear which values are more reliable. The website has an explicit value for CCSD(T)-F12a/cc-pVTZ-F12, but not for other basis sets.